### PR TITLE
GitCommitBear: Improve shortlog message

### PR DESCRIPTION
**Pull request by jarifibrahim**
Fixes https://github.com/coala-analyzer/coala-bears/issues/267

### CODE REVIEW

File: bears/vcs/git/GitCommitBear.py [hunk 1]
```diff
@@ -94,8 +94,12 @@ def check_shortlog(self,
                                          care").
         :param shortlog:                 The shortlog message string.
         """
-        if len(shortlog) > shortlog_length:
-            yield Result(self, "Shortlog of HEAD commit is too long.")
+        diff = len(shortlog) - shortlog_length
+        if diff > 0:
+            yield Result(self,
+                         "Shortlog of HEAD commit is longer than the "
```
Comment (sils1297) [inline bears/vcs/git/GitCommitBear.py]: I'd rather phrase `is {} characters longer`...

Comment (jarifibrahim) [inline bears/vcs/git/GitCommitBear.py]: Done

**Comment by sils1297** [comment]
https://github.com/coala-analyzer/coala-bears/pull/269/commits/085219e224aa1893077ea0fa294ffe11d867756c is ready

**Comment by sils1297** [comment]
https://github.com/coala-analyzer/coala-bears/pull/269/commits/834acd07601b6a53a59884f1fcf3ffcc802f9e18 is great and ready

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**